src/components/site/SubpageTabs.md

```diff
@@ -0,0 +1,23 @@
+### Subpage Tabs
+
+```jsx
+import SubPageTabs from "./SubPageTabs";
+import SubPageTabItem from "./SubPageTabItem";
+import Section from "../basic/containers/Section";
+
+<Section>
+  <SubPageTabs text="In This Section" id="dg_menu-trigger-btn">
+    <SubPageTabItem className="active" text="Newsroom" link="/home" />
+    <SubPageTabItem text="Social Media Gallery" link="/home" />
+    <SubPageTabItem text="Media Contact Information" link="/home" />
+  </SubPageTabs>
+</Section>;
+```
+
+Html Snippet:
+
+```html
+<div class="dark">
+  <ul></ul>
```

Comment:
example looks likes its missing some markup.

Reply:
Setup the .md file while I was fine-tuning the markup and completely forgot to circle back and update this. Added relevant html markup  and committed.